extensions/wikia/UserLogin/tests/UserLoginTest.php

```diff
@@ -512,46 +511,18 @@ public function changePasswordDataProvider() {
 		);
 		$expMsg9 = wfMessage( 'badretype' )->escaped();
 
-		// 10 error --  not match temporary password (checkTemporaryPassword = false)
 		$params10 = array(
 			'username' => 'WikiaUser',
 			'newpassword' => 'testPasword',
 			'retype' => 'testPasword',
 			'loginToken' => self::LOGIN_TOKEN,
 		);
-		$mockUserParams10 = array(
-			'load' => null,
-			'loadFromDatabase' => null,
-			'isAnon' => false,
-			'checkTemporaryPassword' => false,
-			'checkPassword' => true,
-		);
-		$expMsg10 = wfMessage( 'userlogin-error-wrongpassword' )->escaped();
-
-		// 11 error -- not correct password (checkPassword = false)
-		$mockUserParams11 = array(
-			'load' => null,
-			'loadFromDatabase' => null,
-			'isAnon' => false,
-			'checkTemporaryPassword' => true,
-			'checkPassword' => false,
-		);
-
-		// 1011 error -- [10] not match temporary password (checkTemporaryPassword = false) + [11] not correct password (checkPassword = false)
-		$mockUserParams1011 = array(
-			'load' => null,
-			'loadFromDatabase' => null,
-			'isAnon' => false,
-			'checkTemporaryPassword' => false,
-			'checkPassword' => false,
-		);
 
 		// 12 error -- not valid new password (passwordtooshort)
 		$mockUserParams12 = array(
 			'load' => null,
 			'loadFromDatabase' => null,
 			'isAnon' => false,
-			'checkTemporaryPassword' => true,
 			'checkPassword' => true,
 			'getPasswordValidity' => 'passwordtooshort',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "getPasswordValidity" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)